### PR TITLE
[EditorialManager.com] Default_off ruleset

### PR DESCRIPTION
**Pull request by Bisaloo**

### CODE REVIEW

File: src/chrome/content/rules/Editorialmanager.com.xml [hunk 1]
```diff
@@ -1,23 +1,18 @@
 <!--
-	Problematic subdomains:
+	This ruleset currently prevents the download of some pdf.
```
Comment (J0WI) [inline src/chrome/content/rules/Editorialmanager.com.xml]: Can we solve this with exclusions?

File: src/chrome/content/rules/Editorialmanager.com.xml [hunk 2]
```diff
@@ -1,23 +1,18 @@
...
 
-		- ^ ¹
-
-	¹: Not found
-
-	Note: I think that it might also be possible to redirect
-	http://www.edmgr.com/ to https://www.editorialmanager.com.
-	This is left out to be on the safe side.
+	Test http://www.editorialmanager.com/homepage/DOCS/Reviewer_Tutorial.pdf
+	with and without HTTPS-Everywhere enabled.
```
Comment (J0WI) [inline src/chrome/content/rules/Editorialmanager.com.xml]: Wildcard DNS

**Comment by J0WI** [comment]
@Bisaloo are you still active in this PR?

**Comment by Bisaloo** [comment]
Yes. I just need some time because fixing rulesets on broken websites is tricky.

**Comment by J0WI** [comment]
@Bisaloo just a friendly reminder here :)

**Comment by zoracon** [comment]
Closing for now, can reopen if interest to finish is noted.

**Pull request closed without merging**